extending-valtimo/tabs/customizing-case-tabs.md

```diff
@@ -1,4 +1,8 @@
-# Customizing case details tabs
+# ⚠️**WARNING**⚠️
```

Comment:
I'll just post this here, but it goes for every warning: Please use the gitbook recommended way of doing warning hints. See: https://docs.gitbook.com/content-creation/blocks/hint